plugins/uast-kotlin-fir/testData/declaration/importOnDemand.log.txt

```diff
@@ -1,3 +1,3 @@
 UFile (package = declaration)
     UImportStatement (isOnDemand = true)
-    UImportStatement (isOnDemand = true)
+    UImportStatement (isOnDemand = true)
```

Comment:
As expected(?), `UImportStatment` is in its final shape, and indeed dumps are identical. (Changes due to `trim()`.)